packages/core/http/core-http-server-internal/src/cookie_session_storage.ts

```diff
@@ -18,7 +18,7 @@ import type {
 } from '@kbn/core-http-server';
 import { ensureRawRequest } from '@kbn/core-http-router-server-internal';
 
-class ScopedCookieSessionStorage<T extends Record<string, any>> implements SessionStorage<T> {
+class ScopedCookieSessionStorage<T extends object> implements SessionStorage<T> {
```

Comment:
`Record<string, any>` requires an index signature, which is not really required here.